### PR TITLE
feat: create separate task to checkJobStatus

### PR DESCRIPTION
**Pull request by aballiet**
This closes https://github.com/kestra-io/plugin-airbyte/issues/46

### CODE REVIEW

File: src/main/java/io/kestra/plugin/airbyte/connections/CheckStatus.java [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    @PluginProperty
+    @Builder.Default
+    Duration maxDuration = Duration.ofMinutes(60);
+
+    @Builder.Default
+    @Getter(AccessLevel.NONE)
+    private transient Map<Integer, Integer> loggedLine = new HashMap<>();
+
+    @Schema(
+            title = "Specify frequency for sync attempt state check API call"
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/airbyte/connections/CheckStatus.java:77]:
```suggestion
            title = "Specify how often the task should poll for the sync status"
```

**Review by anna-geller — APPROVED** [review]
functionality-wise, LGTM

Thanks for adding this 👍

File: src/main/java/io/kestra/plugin/airbyte/connections/CheckStatus.java [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+import static io.kestra.core.utils.Rethrow.throwSupplier;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+        title = "Check Job status of a running sync connection"
```
Comment (brian-mulier-p) [inline src/main/java/io/kestra/plugin/airbyte/connections/CheckStatus.java:40]:
```suggestion
        title = "Check job status of a running sync connection"
```

**Review by brian-mulier-p — APPROVED** [review]
LGTM

**Pull request merged**